src/main/java/com/hazelcast/aws/AwsProperties.java

```diff
@@ -81,7 +81,7 @@
     CONNECTION_TIMEOUT_SECONDS("connection-timeout-seconds", INTEGER, true),
 
     /**
-     * Number of retries while connecting to AWS Services. Its default value is 10.
+     * Number of retries while connecting to AWS Services. Its default value is 3.
```

Comment:
Are you changing the default value or the documentation was wrong?

Reply:
The documentation was wrong.